src/community/dyndimension/src/main/java/org/geoserver/web/data/resource/DynamicDimensionsTabPanel.java

```diff
@@ -272,12 +273,10 @@ public void validate(IValidatable<DefaultValueConfigurations> validatable) {
 
         @Override
         public void convertInput() {
-            visitChildren(TextArea.class, new org.apache.wicket.Component.IVisitor<TextArea<?>>() {
-
-                @Override
-                public Object component(TextArea<?> component) {
-                    component.updateModel();
-                    return null;
+            visitChildren(TextArea.class, (Component component, final IVisit<Void> visit) -> {
```

Comment:
Converted over all the `(Component component, final IVisit<Void> visit)` use to `(component, visit)` as a new commit.